app/services/early_career_teachers/create.rb

```diff
@@ -44,7 +44,7 @@ def call
 
     attr_reader :full_name, :email, :start_term, :school_cohort, :mentor_profile_id, :year_2020
 
-    def initialize(full_name:, email:, school_cohort:, mentor_profile_id: nil, start_term: "autumn_2021", year_2020: false)
+    def initialize(full_name:, email:, school_cohort:, mentor_profile_id: nil, start_term: "spring_2022", year_2020: false)
       @full_name = full_name
       @email = email
       @start_term = start_term
```

Comment:
Should we initialize `start_term` with nil and then `@start_term = start_term || ParticipantProfile::ECF.start_term_options(school_cohort.cohort.start_year)`?

Reply:
If we are relying on this default (i.e. not supplying a value normally) from calling code then it makes sense to do what you suggest